fdbrpc/fdbrpc.h

```diff
@@ -403,6 +403,7 @@ class RequestStream {
 
 	bool operator == (const RequestStream<T>& rhs) const { return queue == rhs.queue; }
 	bool isEmpty() const { return !queue->isReady(); }
+	uint32_t size() const { return queue->size(); }
```

Comment:
Seems like this is added but not used?

Reply:
yeah, at one point I actually used it, but figured it was still probably a good function so I left it in